### DISCUSSION TITLE
如何实现默认语言，而不转跳zh-CN

### DISCUSSION THREAD

**Discussion by xlenco** [spencerwooo/onedrive-vercel-index#552] [Q&A] [answered]
如何实现默认语言，而不转跳zh-CN，
就是想让exp.com/zh-CN,变成exp.com

**Comment by spencerwooo** [comment, marked as answer]
默认情况下语言是根据你浏览器语言来判断的，关闭自动语言判断可以参考 - [Disabling Automatic Locale Detection](https://nextjs.org/docs/advanced-features/i18n-routing#disabling-automatic-locale-detection)

**Comment by xlenco** [comment]
其实我是想让它的地址为原始地址（exp.com），而不是exp.com/zh-CN

**Reply by naicfeng** [reply]
> 其实我是想让它的地址为原始地址（exp.com），而不是exp.com/zh-CN

https://github.com/spencerwooo/onedrive-vercel-index/blob/f7732101e8e495dbd3d6bfb64c3e3c3fb67a5314/next-i18next.config.js#L5

https://github.com/spencerwooo/onedrive-vercel-index/blob/f7732101e8e495dbd3d6bfb64c3e3c3fb67a5314/components/SwitchLang.tsx#L25

把默认语言改为`zh-CN`

**Reply by xlenco** [reply]
借鉴[zkeq](https://github.com/zkeq)大佬的代码已经实现，不过还是谢谢了
